diff --git a/node_modules/react-native-blind-threshold-bls/android/build.gradle b/node_modules/react-native-blind-threshold-bls/android/build.gradle
index 93c7866..63efe53 100644
--- a/node_modules/react-native-blind-threshold-bls/android/build.gradle
+++ b/node_modules/react-native-blind-threshold-bls/android/build.gradle
@@ -76,6 +76,6 @@ repositories {
 
 dependencies {
     //noinspection GradleDynamicVersion
-    implementation 'com.facebook.react:react-native:+'  // From node_modules
+    implementation 'com.facebook.react:react-native:0.66.4'  // From node_modules
     implementation 'net.java.dev.jna:jna:5.12.1@aar'
 }
